configure.ac

```diff
@@ -75,7 +75,7 @@ AM_PATH_GLIB_2_0(,,AC_MSG_ERROR([GLib not found]))
 dnl When bumping the gio-unix-2.0 dependency (or glib-2.0 in general),
 dnl remember to bump GLIB_VERSION_MIN_REQUIRED and
 dnl GLIB_VERSION_MAX_ALLOWED in Makefile.am
-GIO_DEPENDENCY="gio-unix-2.0 >= 2.40.0"
+GIO_DEPENDENCY="gio-unix-2.0 >= 2.50.0"
```

Comment:
Let's factor this out as a separate commit?  I think the main drag here is [Debian Jessie](https://packages.debian.org/jessie/libglib2.0-0).  Still somewhat old is the [Fedora 24 glib2](https://bodhi.fedoraproject.org/updates/FEDORA-2016-6f929357d6).  I honestly don't care about F24 myself but anyways, let's discuss this in a separate PR?